doc.go

```diff
@@ -124,52 +128,56 @@ responsibility of one of the filters to generate the response.
 
 Route Definitions
 
-Route definitions consist of request matching conditions, optional
-filters and a route backend. The eskip package implements the in-memory
-and text representations of route definitions with a parser.
+Route definitions consist of the following: request matching conditions;
+optional filters; a route backend. The eskip package implements the
+in-memory and text representations of route definitions, including a
+parser.
 
 (Note to contributors: in order to stay compatible with 'go get', the
 generated part of the parser is stored in the repository. When changing
 the grammar, 'go generate' needs to be executed explicitly to update the
 parser.)
 
-For details, see the documentation of the eskip subdirectory.
+For details, see the 'eskip' package documentation
 
 
 Data Sources
 
-Skipper loads the route definitions from one or more sources, and
-receives incremental updates while running. It provides three different
-data clients:
+The route definitions of Skipper are loaded from one or more sources and
+while running, it receives incremental updates from them. It provides
+three different data clients:
 
 - Innkeeper: the Innkeeper service implements a storage for large sets
 of skipper routes, with an HTTP+JSON API, OAuth2 authentication and role
-management. See the innkeeper subdirectory and
+management. See the 'innkeeper' package and
 https://github.com/zalando/innkeeper.
 
 - etcd: skipper can load routes and receive updates from etcd clusters
-(https://github.com/coreos/etcd). See the etcd subdirectory.
+(https://github.com/coreos/etcd). See the 'etcd' package.
 
 - static file: package eskipfile implements a simple data client, which
 can load route definitions from a static file in eskip format.
-Currently, it supports only loading on startup and no updates.
+Currently, it supports only loading the routes on startup but no updates.
```

Comment:
"Currently, it only loads the routes on startup. It doesn't support runtime updates"